### PR TITLE
Onboarding, adding integrations- update

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Onboarding > adding integrations:  update
- to reflect current software  https://github.com/home-assistant/frontend/pull/21447
- change tutorial to use Workday instead of Google Cast
- because it can be used also by people who do not have any devices yet
- add some familiarization steps about pages that can be accessed from the integrations details page

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature others were adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/21447
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Enhanced the tutorial for adding the Workday integration, including clearer navigation and expanded step-by-step instructions.
	- Added visual aids, such as screenshots, to assist users in understanding the integration process.

- **Documentation**
	- Revised the document for improved clarity and consistency, including updated headings and an expanded introduction.

### CODE REVIEW

File: source/getting-started/integration.markdown [hunk 1]
```diff
@@ -1,31 +1,82 @@
...
+   - If devices were discovered in your network, you will see them in the **Discovered** section.
 
-Select **Google Cast** from the list. Because my Chromecast device was already discovered, it asks if I want to add the discovered one or a different one. I will choose the latter and setup another instance.
+   ![Screenshot of the integrations page, with discovered devices](/images/getting-started/integrations_page_discovered.png)
+  
+2. If there are any devices discovered for you, you can add them now.
+   - Under **Discovered**, on the integration, select **Configure**.
+   - Follow the steps in the UI, if additional configuration is required.
+   - If you do not see any devices discovered, don't worry, we will add an integration in the next step.
+3. In the bottom right corner, select **Add integration**.
```
Comment (c0ffeeca7) [inline source/getting-started/integration.markdown]:
```suggestion
3. In the bottom right-corner, select **Add integration**.
```

File: source/getting-started/integration.markdown [hunk 2]
```diff
@@ -1,31 +1,82 @@
...
+
+   ![Screenshot of the workday sensor info page](/images/getting-started/workday_service_info.png)
+3. Select **Entity**, to open the list of entities.
+   - Although your Home Assistant already has many entities, you see only 1 entry in the entities table.
+   - This is because it is [filtered](/docs/organizing/tables#filtering-items-in-a-table).
+   - You see the entity ID, which consist of the domain (a technical term for device type) and the entity name, in the following format: `domain.entity_name`.
+
+   ![Screenshot of the entities list, filtered for workday](/images/getting-started/workday_entities_list.png)
+
+## Making changes to the integration
```
Comment (c0ffeeca7) [inline source/getting-started/integration.markdown]:
```suggestion
## Modifying the integration
```

File: source/getting-started/integration.markdown [hunk 3]
```diff
@@ -1,31 +1,82 @@
...
+1. To change the name, select the three dots {% icon "mdi:dots-vertical" %} menu, and select **Rename**.
+
+   ![Screenshot of the entities list, filtered for workday](/images/getting-started/workday_three_dots.png)
+2. You can also add another Workday sensor. For example, to check if tomorrow is a workday.
+   - Select **Add entry**, give it a name and define your options.
+   - To check if tomorrow is a workday, under **Offset**, enter `1`.
+   - You can also add different sensors for different countries, for example if you want to know when your colleagues have a holiday.
+3. That's it! {% icon "mdi:party-popper" %}
+   - {% icon "mdi:checkbox-outline" %} You now have an overview of the integration page, and know there to find the integration details page, the sensor info page, and the entities table.
+   - {% icon "mdi:checkbox-outline" %} You learned how to rename your integration, how to make changes to it and how to delete it.
```
Comment (c0ffeeca7) [inline source/getting-started/integration.markdown]:
```suggestion
   - {% icon "mdi:checkbox-outline" %} You learned to rename, to modify, and how to delete an integration.
```

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**